src/main/telemetry/crsf.c

```diff
@@ -350,7 +351,9 @@ void initCrsfTelemetry(void)
 #endif
 
     int index = 0;
-    crsfSchedule[index++] = BV(CRSF_FRAME_ATTITUDE_INDEX);
+    if (accelerometerConfig()->acc_hardware != ACC_NONE) {
```

Comment:
@codecae  I think ` if (sensors(SENSOR_ACC)) {` look better and you will not need extra include